Config/envoy.yaml

```
@@ -0,0 +1,74 @@
+admin: # 管理者画面の設定
+  access_log_path: /tmp/admin_access.log # ログを保存するファイル
+  address:
+    socket_address: { address: 0.0.0.0, port_value: 9901 } # 9901番ポートでlisten
+
+static_resources:
+  listeners:
+    - name: listener_0
+      address:
+        socket_address: { address: 0.0.0.0, port_value: 50020 } # 51020番ポートでlisten
+      filter_chains: # リクエストのフィルタリングの設定
+        - filters:
+            - name: envoy.filters.network.http_connection_manager # HTTP接続用に設計された組み込みフィルター
+              typed_config:
+                "@type": type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager
+                codec_type: auto
+                stat_prefix: ingress_http # 接続マネージャの統計を発行する時に使用する人間が読める形式のprefix
+                access_log:
+                  - name: envoy.access_loggers.file
+                    typed_config:
+                      "@type": type.googleapis.com/envoy.extensions.access_loggers.file.v3.FileAccessLog
+                      path: "/dev/stdout" # ログは標準出力
+                route_config:
+                  name: local_route
+                  virtual_hosts:
+                    - name: local_service
+                      domains: ["*"] # virtual_hostのドメイン
+                      routes:
+                        - match: { prefix: "/" } # "/"がリクエストのルートと一致
+                          route:
+                            cluster: cc_to_bts
+                      cors:
+                        allow_origin_string_match: # 許可されたorigin
+                          - prefix: "*"
+                        allow_methods: GET, PUT, DELETE, POST, OPTIONS
+                        allow_headers: keep-alive,user-agent,cache-control,content-type,content-transfer-encoding,custom-header-1,x-accept-content-transfer-encoding,x-accept-response-streaming,x-user-agent,x-grpc-web,grpc-timeout
+                        max_age: "1728000" # access-control-max-age headerを指定
+                        expose_headers: custom-header-1,grpc-status,grpc-message # access-control-expose-headers headerを指定
+                http_filters:
+                  - name: envoy.filters.http.grpc_web # gRPC-Webクライアントを準拠したgRPCサーバーにブリッジできるようにするフィルター
+                  - name: envoy.filters.http.cors
+                  - name: envoy.filters.http.router
+                use_remote_address: true
+                skip_xff_append: false
+      # transport_socket:
```

Please add comment for later edits if you don't mind.

例:

```suggestion
      # TLS を用いて通信を行う場合コメントを外す
      # transport_socket:
```

---

@arukuka 
Thanks for the review!
I fixed it with [785fdff](https://github.com/acompany-develop/QuickMPC-BTS/pull/7/commits/785fdff4b5f5854970c8cc0f37a795b8a4f63360)